pkg/utils/validator/clusterresourceplacement_test.go

```diff
@@ -190,7 +190,7 @@ func TestValidateClusterResourcePlacementAlpha(t *testing.T) {
 				t.Errorf("ValidateClusterResourcePlacementAlpha() error = %v, wantErr %v", gotErr, testCase.wantErr)
 			}
 			if testCase.wantErr && !strings.Contains(gotErr.Error(), testCase.wantErrMsg) {
-				t.Errorf("ValidateClusterResourcePlacementAlpha() failed to find expected error message = %s, in error = %s", testCase.wantErrMsg, gotErr.Error())
+				t.Errorf("ValidateClusterResourcePlacementAlpha() failed to find expected error message = %v, in error = %v", testCase.wantErrMsg, gotErr.Error())
```

Comment:
```suggestion
				t.Errorf("ValidateClusterResourcePlacementAlpha() got %v, want contain %s", , gotErr, testCase.wantErrMsg)
```
got before want, no need to call goErr.Error()

https://go.dev/wiki/TestComments#got-before-want